Incorrect detection of strings with multiple %[1]

https://github.com/catenacyber/perfsprint/pull/16 fails to consider cases where the one arguments is referenced multiple times:

```golang
package main

import "fmt"

func main() {
         _ = fmt.Sprintf("You say %[1]s, I also say %[1]s", "hello")
}
```

Using the version before the PR mentioned above, there is no change as positional arguments weren't supported.
```
$ go install github.com/catenacyber/perfsprint@v0.4.0
$ perfsprint --fix ./...
```

Using a the latest version (i.e. after the PR was merged) it detects and gives an inccorrect suggested "fix":
```console
$ go install github.com/catenacyber/perfsprint@v0.7.0
$ perfsprint --fix ./...
test/main.go:6:7: fmt.Sprintf can be replaced with string concatenation
test/main.go:3:8: Fix imports
````

The fix is:
```
package main

import

func main() {
         _ = "You say %[1]s, I also say "+"hello"
}
```

Given this is not a simple concatenation I would expect this to just not be flagged by perfsprint.

--- a/analyzer/analyzer.go
+++ b/analyzer/analyzer.go
@@ -62,6 +62,9 @@ func isConcatable(verb string) bool {
 		(strings.HasSuffix(verb, "%s") && !strings.Contains(verb, "%[1]s")) ||
 			(strings.HasSuffix(verb, "%[1]s") && !strings.Contains(verb, "%s"))
 
+	if strings.Count(verb, "%[1]s") > 1 {
+		return false
+	}
 	return (hasPrefix || hasSuffix) && !(hasPrefix && hasSuffix)
 }
 

--- a/analyzer/testdata/src/p/p.go
+++ b/analyzer/testdata/src/p/p.go
@@ -212,6 +212,7 @@ func negative() {
 	fmt.Sprintf("% d", 42)
 	fmt.Sprintf("%-10d", 42)
 	fmt.Sprintf("%s %[1]s", "hello")
+	fmt.Sprintf("%[1]s %[1]s", "hello")
 	fmt.Sprintf("%[2]d %[1]d\n", 11, 22)
 	fmt.Sprintf("%[3]*.[2]*[1]f", 12.0, 2, 6)
 	fmt.Sprintf("%d %d %#[1]x %#x", 16, 17)

--- a/analyzer/testdata/src/p/p.go.golden
+++ b/analyzer/testdata/src/p/p.go.golden
@@ -214,6 +214,7 @@ func negative() {
 	fmt.Sprintf("% d", 42)
 	fmt.Sprintf("%-10d", 42)
 	fmt.Sprintf("%s %[1]s", "hello")
+	fmt.Sprintf("%[1]s %[1]s", "hello")
 	fmt.Sprintf("%[2]d %[1]d\n", 11, 22)
 	fmt.Sprintf("%[3]*.[2]*[1]f", 12.0, 2, 6)
 	fmt.Sprintf("%d %d %#[1]x %#x", 16, 17)

